packages/config/src/options.js

```diff
@@ -473,5 +474,19 @@ export function getNuxtConfig (_options) {
     staticAssets.versionBase = urlJoin(staticAssets.base, staticAssets.version)
   }
 
+  // Nuxt Telemetry
+  if (
+    options.telemetry !== false &&
+    !options.test &&
+    !destr(process.env.NUXT_TELEMETRY_DISABLED) &&
```

Comment:
Do we need a new dependecy `destr` here ?

Reply:
Dependency is added. It is to check for `true` value not just env existence.

Reply:
I mean do we need this dependency for just one checking here ?

Reply:
No, actually I created it to use in more places including runtimeConfig (convert `"true"` to `true`) :see_no_evil: